src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -318,7 +318,8 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
         }
       } catch (Exception e) {
         versionString = LIBRARY_VERSION
-        LOG.warn("An error occurred while trying to determine Checker Framework version. Assuming default is being used: {}. Error caused by: {}", versionString, e.toString())
+        LOG.warn("No explicit dependency on the Checker Framework found, so using plugin default version: {}", LIBRARY_VERSION)
```

Comment:
I would drop "plugin" because it's the CF version, not the plugin version.  (It's the plugin's default CF version, but we don't want to be so verbose.)